package-lock.json

```diff
@@ -5864,36 +5864,16 @@
                 "@types/node": "*"
             }
         },
-        "node_modules/@types/eslint": {
```

Comment:
What are we using in place of eslint?

Reply:
This project is still using eslint. This project's direct dependencies are defined in `package.json`.

This change is pulling in webpack's changes to their dependencies, where they removed `eslint-scope`: #338